apollo-gradle-plugin/src/main/java/com/apollographql/android/gradle/ApolloCodegenArgs.java

```diff
@@ -0,0 +1,31 @@
+package com.apollographql.android.gradle;
+
+import java.io.File;
+import java.util.Set;
+
+
+public class ApolloCodegenArgs {
+  private File schemaFile;
+  private Set<String> queryFiles;
+
+  public ApolloCodegenArgs(File schema, Set<String> queries) {
```

Comment:
Does this class need both a constructor and setters? Seems like a value object, could we make immutable